src/testing/index.ts

```diff
@@ -58,9 +58,9 @@ export async function initialise(context: ExtensionContext) {
     await commands.executeCommand(`setContext`, `projectExplorer:testing`, true);
     const ibmi = getInstance()!;
     if (!individualTesting) {
-      ibmi.onEvent(`connected`, runTests);
+      ibmi.subscribe(context, `connected`, "Run IBM i Project Explorer tests", runTests);
```

Comment:
Can all these strings be wrapped in `l10n.t` and then run `npm run nls` to extract them?

Reply:
Sure thing!